store/src/main/java/com/nytimes/android/external/store/util/NoopPersister.java

```diff
@@ -35,4 +37,10 @@
     public void clear(@Nonnull Key key) {
         networkResponses.remove(key);
     }
+
+    @Nonnull
+    @Override
+    public Observable<Raw> readAll(@Nonnull Key key) throws FileNotFoundException {
+        return null;
```

Comment:
I'm not a fan of returning null here especially when Rxjava2 does not allow you to propogate a null to a consumer.  Ideally we can structure the base classes in a manner that would not need this override. If that is too messy then throw an exception from here instead.